benchmark/dbally_benchmark/e2e_benchmark.py

```diff
@@ -82,10 +83,13 @@ async def evaluate(cfg: DictConfig) -> Any:
 
     engine = create_engine(benchmark_cfg.pg_connection_string + f"/{cfg.db_name}")
 
-    llm = LiteLLM(
-        model_name="gpt-4",
-        api_key=benchmark_cfg.openai_api_key,
-    )
+    if "gpt" in cfg.model_name:
```

Comment:
Maybe we could structure this configuration variable in the same way that LiteLLM does for selecting model providers (e.g., anyscale/mistralai/Mistral-7B-Instruct-v0.1, azure/<model_name>, etc.).

In our case, we would have something like:

```python
if cfg.model_name.startswith("local/"):
     llm = LocalLLM(model_name= cfg.model_name.split("/", 1)[1], ...)
else:
    llm = LiteLLM(...)
```